issue_16/main.js

```diff
@@ -0,0 +1,52 @@
+const ul = document.getElementById('js-ul');
+const endpoint = 'https://api.json-generator.com/templates/5-2NhpTqkaV4/data?access_token=mihb6ib9irdyvy1iwbqg804flfdpm6kxdwtoum6x';
+
+function createNewElements(tagName, className) {
+    const newElement = document.createElement(tagName);
+    newElement.classList.add(className);
+    return newElement;
+}
+
+function displayErrorMessage() {
+    const errorMessage = createNewElements('p', 'error-message');
+    errorMessage.textContent = 'Communication failed on the server side.';
+    ul.appendChild(errorMessage);
+}
+
+async function fetchData() {
+    try {
+        const response = await fetch(endpoint);
+        if (!response.ok) {
+            console.error(`${response.status}:${response.statusText}`);
+        }
+        const result = await response.json();
+        return result.data;
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+
+const renderTabs = (tabElements) => {
+    const fragment = new DocumentFragment();
+    for (const tabElement of tabElements) {
+        const tabList = createNewElements('li', 'tab-list');
+        tabList.textContent = tabElement.category;
+        fragment.appendChild(tabList);
+    }
+    ul.appendChild(fragment);
+}
+
+async function init() {
+    let resultOfFetchData;
```

Comment:
nits
わかるので
```suggestion
    let res
```
でも良いです

Reply:
https://github.com/yoshikawa-akr/js-lessons/pull/17/commits/d988da4b1c7cd9613339b7af6124adb14b758fc5
fixed.